.mailmap

```diff
@@ -329,6 +334,7 @@ San-Tai Hsu <v@fatpipi.com>
 Santiago Gimeno <santiago.gimeno@gmail.com> <santiago.gimeno@ionide.es>
 Sarah Meyer <sarahsaltrick@gmail.com> sarahmeyer <sarahsaltrick@gmail.com>
 Sartrey Lee <sartrey@163.com> sartrey <sartrey@163.com>
+Saúl Ibarra Corretgé <s@saghul.net> <saghul@gmail.com>
```

Comment:
@saghul Is the saghul.net one the correct one? That’s what this is assuming here but please correct me if I’m wrong.

Reply:
Yeah, that's the one I use these days, but I do have commits with the old (gmail) one from the past. I'd rather use the new one.